tests/test_jx/test_metadata.py

```diff
@@ -111,7 +111,7 @@ def test_get_nested_columns(self):
                         {"b": "x", "v": 2},
                         {"b": "y", "v": 3}
                     ]},
-                    {"o": 2, "_a": {"b": "x", "v": 5}},
+                    {"o": 2, "_a": [{"b": "x", "v": 5}]},
```

Comment:
As per schema expansion, Change in  an inner object to nested array of objects.
I hope my assumptions are correct here.